vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -848,6 +848,15 @@
     src: server.yml
     dest: /home/arkcase/.arkcase/acm/acm-config-server-repo/arkcase-server.yaml
 
+- name: server-specific configuration for arkcase-commons.yml for version 2021.04
```

Comment:
`2021.04` ???  should it be '2022.01` ?

Reply:
It works the same. We started with 2021.04 and didn't want to change everywhere.